src/mca/schizo/prte/schizo_prte.c

```diff
@@ -1111,13 +1111,15 @@ static void allow_run_as_root(prte_cli_result_t *cli)
 {
     char *r1, *r2;
 
-    if (prte_cmd_line_is_taken(cli, "allow-run-as-root")) {
+    if (pmix_cmd_line_is_taken(cli, "allow-run-as-root")) {
```

Comment:
same here.

Reply:
Wow - that is strange that it compiled and completed ci.

Thanks, will fix